app/models/dictionary.rb

```diff
@@ -176,6 +176,10 @@ def num_black
 		entries.where(mode:Entry::MODE_BLACK).count
 	end
 
+  def num_auto_expanded
+    entries.where(mode:Entry::MODE_AUTO_EXPANDED).count
```

Comment:
```suggestion
    entries.auto_expanded.count
```

今回追加した、scopeをつかって上のように書けそうです。
既存に合わせてくれたと思いますが、既存も直してしまいたいです。
おそらく画面に表示された数値をみれば、比較的簡単に動作確認できると思っています。

またメソッドの本体が一行だけの場合は次のように書けます。
試してみてください。

```ruby
def num_auto_expanded = entries.auto_expanded.coun
```

Reply:
ありがとうございます。
[count定義の書き方を変更](https://github.com/pubannotation/pubdictionaries/pull/73/commits/7ab06479bf49f0b467592052fd5cab4410908804)にて修正し、値も問題なく反映されていることを確認しました。